src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -36,31 +39,78 @@ import scala.annotation.tailrec
  * For more details @see https://en.wikipedia.org/wiki/Kolakoski_sequence
  */
 
-object Homework :
+object Homework:
 
-  object `Boolean Operators` :
+  object `Boolean Operators`:
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = if (b) false else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean = if (left) if (right) true else false else false

```

Comment:
This could be simplified without nested `if`s.

Reply:
Make sure that `and` works like Java's `&&` means that if `left` is false then `right` is not calculated. Provide tests for this case.